python/pyspark/pandas/base.py

```diff
@@ -873,7 +873,11 @@ def isin(self: IndexOpsLike, values: Sequence[Any]) -> IndexOpsLike:
             )
 
         values = values.tolist() if isinstance(values, np.ndarray) else list(values)
-        return self._with_new_scol(self.spark.column.isin([SF.lit(v) for v in values]))
+
+        other = [SF.lit(v) for v in values]
+        scol = self.spark.column.isin(other)
+
+        return self._with_new_scol(F.when(scol.isNull(), False).otherwise(scol))
```

Comment:
I guess `F.coalesce` is better? `F.coalesce(scol, F.lit(False))`

Reply:
Btw, we can mange `field` here?
`field=self._internal.data_fields[0].copy(dtype=np.dtype("bool"), spark_type=BooleanType(), nullable=False)`

Reply:
thank you, done.